workspaces/linguist/plugins/linguist-backend/src/schema/openapi.yaml

```diff
@@ -0,0 +1,109 @@
+openapi: '3.0.3'
+info:
+  title: linguist
+  description: This API powers the backend for the linguist plugins of backstage.
+  version: '1.0'
+  license:
+    name: Apache-2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  contact: {}
+servers:
+  - url: /
+paths:
+  /health:
+    get:
+      description: Checks if the linguist backend is hooked up properly
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /entity-languages:
+    get:
+      description: Returns the language breakdown of the passed in entityRef.
+      parameters:
+        - in: query
+          description: Reference passed in the format <kind>:<namespace>/<name>
+          example: component:default/my-component
+          name: entityRef
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Languages'
+        '500':
+          $ref: '#/components/responses/ServerError'
+components:
+  examples: {}
+  headers: {}
+  parameters: {}
+  requestBodies: {}
+  responses:
+    ServerError:
+      description: Error
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              error:
+                type: object
+                properties:
+                  name:
+                    type: string
+                    example: Error
+                  message:
+                    type: string
+                    example: No entityRef was provided
+  schemas:
+    LanguageType:
```

Comment:
@aramissennyeydd , just curious but is there a convention where we need to prefix our schemas with the plugin name so that there aren't name conflicts in the stitched together spec? Or will you be doing that automatically?

Reply:
Does this need to be resolved to move forward?

Reply:
As far as I can tell, this is a future issue that needs to have some planning / validation put in place to prevent problems. Likely the validation tools that @aramissennyeydd should be enforcing a naming convention for the schemas here, similar to how it requires the api paths to be prefixed with the plugin name.

We could always get ahead of that and add a namespace to these schemas here, but there needs to be a larger convention put in place that all plugins follow.. or the schema stitching needs to edit the object names to prevent conflicts.

Reply:
Ok, let's leave it for a follow up PR then in the new year